modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp

```diff
@@ -99,9 +95,44 @@ portletDisplay.setURLBack(backURL);
 						{}
 					);
 
-					ddmFormValues.value = JSON.stringify(values);
-
-					Liferay.Util.submitForm(form);
+					Liferay.Util.fetch(
+						'/o<%= objectDefinition.getRESTContextPath() %>',
```

Comment:
`objectDefinition.getRESTContextPath()` returns the path related to current endpoint?

Reply:
Yes, @interaminense!